CHANGES.md

```diff
@@ -4,6 +4,9 @@
 
 #### _Black_
 
+- `Black` now remove leading non-breaking spaces in comments before adding a space
```

Comment:
```suggestion
- `Black` now cleans up leading non-breaking spaces in comments
```
I think this is better; we don't need to be too specific about the exact output.